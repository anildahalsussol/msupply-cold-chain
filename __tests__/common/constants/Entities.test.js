import { ENTITIES } from '~constants';

describe('Constants: ENTITIES', () => {
  it('Matches snapshot', () => {
    expect(ENTITIES).toMatchSnapshot();
  });
});
