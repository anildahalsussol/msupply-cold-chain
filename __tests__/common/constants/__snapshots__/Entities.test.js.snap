// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Constants: ENTITIES Matches snapshot 1`] = `
Object {
  "SENSOR": "Sensor",
  "SENSOR_LOG": "SensorLog",
  "SETTING": "Setting",
  "SYNC_CONFIG": "SyncConfig",
  "SYNC_LOG": "SyncLog",
  "TEMPERATURE_BREACH": "TemperatureBreach",
  "TEMPERATURE_BREACH_CONFIGURATION": "TemperatureBreachConfiguration",
  "TEMPERATURE_LOG": "TemperatureLog",
}
`;
